.github/scripts/verifyActions.sh

```diff
@@ -8,16 +8,40 @@ declare -r GREEN='\033[0;32m'
 declare -r RED='\033[0;31m'
 declare -r NC='\033[0m'
 
+# In order for this script to be safely run from anywhere, we cannot use the raw relative path '../scripts'
+declare SCRIPTS_DIR
+SCRIPTS_DIR="$(dirname "$(dirname "$0")")/scripts"
+
+declare LIB_PATH
+LIB_PATH="$(dirname "$(dirname "$0")")/temp/diff-so-fancy"
+
+# Clone diff-so-fancy repository
+if [ ! -d "$LIB_PATH/" ]; then
+    git clone https://github.com/so-fancy/diff-so-fancy.git $LIB_PATH
```

Comment:
cloning a repo on every push seems like a lot. 

Did you try just setting the git config.ui setting to auto to see if we could just use the default git coloring?

Even though this is using Mocha which we don't it seems like it should be supported without the use of an external library https://github.community/t/ansi-color-output-in-webview/17621

https://unix.stackexchange.com/questions/44266/how-to-colorize-output-of-git

so I think we just need a 
```
git config --global color.ui auto
```

Reply:
Hi @bondydaa,

Thank you for reviewing this pull request. I proceed as agreed in the proposal at https://github.com/Expensify/Expensify.cash/issues/1665 . In the proposal I explained that this what I was intended to do. To change that now I would have to undo everything I did and start from scratch.

Are these solutions giving the diffs at _character level_ as well? From what I saw it didn't seem so...

Reply:
@FredericoGauz, please understand that things can change during reviews. Different people can have varying perspectives on issues, and we appreciate some degree of flexibility with our contributors.

That said, I think @bondydaa's suggestions would not produce a character-level diff like we want here. However, maybe you could greatly simplify this by just adding `diff-so-fancy` as a dev dependency. Since `npm ci` is run in the `verifyGithubActionsBuild` workflow _before_ this script is run, that should make it available on the runner (just like how we can use `ncc`). Does that make sense?

Reply:
👍 @roryabraham explained to me why the character level diffing was important, sorry about that!.

Reply:
Hi @bondydaa  and @roryabraham ,

I totally understand that, sorry about this misunderstanding. Looking at my answer again I can see it looks a bit too terse. I didn't mean that at all!

My idea was to separate the dependencies in this workflow from the rest of the application. As it was only used here, and nowhere else.

My logic was as follows:

The github action firsts spins a ubuntu server, then adds ands installs all npm packages  and only then runs the verifyActions step.

The `git clone` command is extremely fast and the library is tiny.  It takes less then a second to clone, since it is just a few files and to tell you the truth I had to play close attention while I was running my tests because I almost always could not see the cloning.

Github Also is also greatly optimized for this use cases and I even added a check to see if the library is already there if you perhaps wanted to run locally.

I would greatly appreciate if you could take my logic into appreciation. Let me know if you want to go forward adding this lib to dev dependencies, although I can't see why it would make much of a difference. All in all I appreciate this is your project and it should follow your own considerations. I am just giving you my suggestions as if I was a part of your inhouse team.

Thanks,

Reply:
I think moving it as a dev dependency is probably the better route, that way anyone can also utilize the library locally if they need to get character level diffing and won't have to hunt for what package to install themselves to achieve it.

Reply:
Hi @roryabraham and @bondydaa,

I was going to reply after making the changes you request. I understand what you both said and agree that it will make it much easier to reuse it elsewhere. It is difficult to make decisions on other people code so I just wanted to wait for your feedback before applying the changes!